### PR TITLE
Replace String.Format with string interpolation

### PR DESCRIPTION
**Pull request by mikkelbu**
Relates to #9002

### CODE REVIEW

File: docs/csharp/programming-guide/concepts/linq/codesnippet/CSharp/data-transformations-with-linq_4.cs [hunk 1]
```diff
@@ -8,7 +8,7 @@ static void Main()
             // Query.
             IEnumerable<string> query =
                 from rad in radii
-                select String.Format("Area = {0}", (rad * rad) * 3.14);
+                select $"Area = {(rad * rad) * 3.14}";
```
Comment (pkulikov) [inline docs/csharp/programming-guide/concepts/linq/codesnippet/CSharp/data-transformations-with-linq_4.cs]: nit picking from math: replace `3.14` with `System.Math.PI`? (or with adding `using System;`)

**Review by pkulikov — COMMENTED** [review]
@mikkelbu nice updates; I've left a couple of minor nits

**Comment by mikkelbu** [comment]
Note that I have not updated the sample from https://code.msdn.microsoft.com/Async-Fine-Tuning-Your-a676abea as I don't know where to access this code.

File: docs/csharp/programming-guide/concepts/linq/codesnippet/CSharp/data-transformations-with-linq_4.cs (hunk 1 above)
Comment (mikkelbu) [reply docs/csharp/programming-guide/concepts/linq/codesnippet/CSharp/data-transformations-with-linq_4.cs]: @pkulikov If I make the change, then I'll also need to update the expected result (in the bottom of the file). Will it be ok to format the `area` to two decimals in the text, if I use `Math.PI`?

Comment (pkulikov) [reply docs/csharp/programming-guide/concepts/linq/codesnippet/CSharp/data-transformations-with-linq_4.cs]: @mikkelbu that's a good point.

I've just checked: if the output is formatted to two decimals:
```csharp
select $"Area = {rad * rad * Math.PI:F2}";
```

the output slightly changes:
```
Area = 3.14
Area = 12.57
Area = 28.27
```

**Review by richlander — APPROVED** [review]
LGTM

**Comment by BillWagner** [comment]
Thanks again @mikkelbu 
I'll :shipit: now.

**Pull request merged**